src/main/java/com/example/MyBatisMember/MemberMapper.java

```diff
@@ -5,9 +5,14 @@
 
 import java.util.List;
 
-
 @Mapper
 public interface MemberMapper {
-    @Select("SELECT * FROM mebers")
+    @Select("SELECT * FROM members")
     List<Member> findAll();
-}
+
+    @Select("SELECT name FROM members")
+    List<String> findAllName();
+
+    @Select("SELECT * FROM members WHERE age >= #{age}")
+    List<Member> findOverAge(String age);
```

Comment:
より大きいを表すなら
```suggestion
    List<Member> findByAgeMoreThan(String age);
```
とするのも良いです！